_posts/2018-11-19-open-core-engineering-group-meeting.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Core Engineering online meeting notes, 19 November 2018
+subtitle: Meeting notes
+---
+
+## Core Engineering online meeting notes, 19 November 2018
+
+This online session 18 people joined to discuss the term "Core Engineering".
+
+Questions could be posted during the meeting with prefix "QUESTION:" and as it was an open session also comments could be added with the "COMMENT:" prefix.
+
+The discussion started with a reference to the [google-document](https://docs.google.com/document/d/1wAAR0v1dglXXQThqaMN_iNHi5xSH1boAlP3q-_4kSWI/edit?usp=sharing) to explain the reason why the discussion is happening. Afterwards Don Syme stated the goals for the meeting being to come up with a list of actionable improvements that the community can make to ease development in the core F# repositories and document those in the github repository Microsoft/visualfsharp or elsewhere.
+
+The particular proposed solutions / taks named Proposed Core Engineering Task (PCET) can be found at the end of the document.
+
+The list of questions and ideas of the google document where worked through in the whole meeting starting with:
```

Comment:
Mention the authors of the doc here, thanks

Reply:
OK, wasn't sure whether it's appropiate to mention me as author as I'm already writing the meeting notes.

Reply:
It's best to write it as if there were two separate voices (two "hats") - one as the meeting note taker, one as yourself